examples/Multi_Armed_Bandit/__init__.py

```diff
@@ -0,0 +1 @@
+"""Multi-armed bandit examples."""
```

Comment:
@julianStreibel @AdrianSosic just sharing the following ICML paper here with you in case this is interesting for this or a future PR in this direction: https://proceedings.mlr.press/v235/jun24a.html